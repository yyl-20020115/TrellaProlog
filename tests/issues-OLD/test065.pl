:- initialization(main).

:- use_module(library(lists)).

main :-
	setof(I, member(I, [A,B,B,A]), Set), writeq(Set), nl,
	bagof(I, member(I, [A,B,B,A]), Bag), writeq(Bag), nl.
