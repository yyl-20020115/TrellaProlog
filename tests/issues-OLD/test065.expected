[_4,_5]
[_4,_5,_5,_4]
